client/securedrop_client/logic.py

```diff
@@ -963,7 +979,15 @@ def on_file_download_success(self, uuid: str) -> None:
         Called when a file has downloaded.
         """
         self.session.commit()
-        file_obj = storage.get_file(self.session, uuid)
+        try:
+            file_obj = storage.get_file(self.session, uuid)
+
+        except storage.SDDatabaseError as e:
+            # This shouldn't happen, it's been downloaded.
+            logger.error("Failed to find file uuid in database")
```

Comment:
Let's have this explain your comment? e.g. "Failed to find file that was just downloaded in the database."

Reply:
Thank you! Addressed in https://github.com/freedomofpress/securedrop-client/commit/138643be480dd82bb9ed24b787a04307a8d8c1ae